go.mod

```diff
@@ -175,3 +175,4 @@ replace github.com/kubernetes/cri-api => k8s.io/cri-api v0.23.5-rc.0
 
 replace github.com/aquasecurity/tracee/types => ./types
 
+replace github.com/aquasecurity/libbpfgo => ../libbpfgo
```

Comment:
This commit will be dropped when libbpfgo bump (with map of maps) gets merged.

Reply:
This depends on https://github.com/aquasecurity/libbpfgo/pull/356 and https://github.com/aquasecurity/libbpfgo/pull/366.

Reply:
https://github.com/aquasecurity/libbpfgo/pull/356 was merged. It's now depending only on https://github.com/aquasecurity/libbpfgo/pull/366 merge.

Reply:
All merged. Raising a change to get the upstream libbpfgo.